doc/rados/operations/pools.rst

```diff
@@ -188,7 +188,7 @@ consider removing them when you no longer need your pool::
 
 	ceph osd pool get {pool-name} crush_ruleset
 
-If the ruleset was "123", for example, you can check the other pools like so::
+If the ruleset ID was "123", for example, you can check the other pools like so::
```

Comment:
Actually, we don't have enough documents on "ceph osd crush rule ..." operating topic, these commands take ruleset name as argument, so I think it's necessary to distinguish "name" and "id".